### PR TITLE
Add react router

### PR DESCRIPTION
**Pull request by JakubZojdzik**
Nothing interesting xd

### CODE REVIEW

File: apps/client/package.json [hunk 1]
```diff
@@ -21,8 +21,11 @@
         "@mui/material": "^5.10.7",
         "@szklarnia-pwr/database": "*",
         "@szklarnia-pwr/dto": "*",
+        "add": "^2.0.6",
```
Comment (maciejopalinski) [inline apps/client/package.json]: Is this accidental?

File: apps/client/package.json [hunk 2]
```diff
@@ -21,8 +21,11 @@
...
         "react": "^18.0.0",
-        "react-dom": "^18.0.0"
+        "react-dom": "^18.0.0",
+        "react-router-dom": "6",
+        "yarn": "^1.22.19"
```
Comment (maciejopalinski) [inline apps/client/package.json]: And this?

**Review by maciejopalinski — DISMISSED** [review]
Approved, but don't merge before fixing those weird packages

**Review by maciejopalinski — APPROVED** [review]
Looks good to me!

File: apps/client/package.json (hunk 2 above)
Comment (JakubZojdzik) [reply apps/client/package.json]: of course xd

**Pull request merged**